### PR TITLE
feat: add ContainsSlice()

### PR DESCRIPTION
**Pull request by ArielHAlba**
https://app.asana.com/0/538204993432209/1117184718515417/f

Added ContainsSlice() function that returns a boolean to if slice contains s

### CODE REVIEW

File: contains.go [hunk 1]
```diff
@@ -68,3 +69,28 @@ func StringIndexContainingSubString(s string, ls ...string) int {
 	}
 	return -1
 }
+
+func stringApplyTransforms(s string, transforms []Transform) string {
```
Comment (ferhatelmas) [inline contains.go]: use same api with slice version

File: contains.go [hunk 2]
```diff
@@ -68,3 +69,28 @@ func StringIndexContainingSubString(s string, ls ...string) int {
...
+
+func sliceApplyTransforms(slice []string, transforms ...Transform) []string {
+	transformed := make([]string, len(slice))
+	for i := range slice {
+		transformed[i] = stringApplyTransforms(slice[i], transforms)
+	}
+	return transformed
+}
+
+// ContainsSlice returns a boolean to know if `slice` contains `s`
```
Comment (ferhatelmas) [inline contains.go]: returns a boolean to know if

checks if

File: contains.go (hunk 1 above)
Comment (ferhatelmas) [reply contains.go]:
```diff
- []Transform
+ ...Transform
```

File: contains.go [hunk 3]
```diff
@@ -68,3 +69,28 @@ func StringIndexContainingSubString(s string, ls ...string) int {
...
+func ContainsSlice(slice [][]string, s []string, transform ...Transform) bool {
```
Comment (ferhatelmas) [inline contains.go]: to be aligned with other functions

seems name would be better like `SliceContains`

File: contains.go [hunk 4]
```diff
@@ -68,3 +69,28 @@ func StringIndexContainingSubString(s string, ls ...string) int {
...
 }
+
+func stringApplyTransforms(s string, transforms []Transform) string {
+	for _, t := range transforms {
+		s = t(s)
+	}
+	return s
+}
+
+func sliceApplyTransforms(slice []string, transforms ...Transform) []string {
```
Comment (ashishbhate) [inline contains.go]: Non-blocking comment

According to naming convention in the file, this should just be `applyTransforms` and the string version should be `stringApplyTransform`. 

The naming in `agstring` is quite messy, so I think we should slowly bring order and consistency to it.

**Pull request merged**